packages/core/src/Card/Card.test.tsx

```diff
@@ -23,7 +24,7 @@ const CardElement = (
 
 test("Renders and matches snapshot", () => {
   const { container } = render(CardElement);
-  expect(container).toMatchSnapshot();
+  expect(container).toBeTruthy();
```

Comment:
This kind of defeats the test, unfortunately. `container` is the DOM representation created by the `render` function, so it will nearly always pass this test. The intention is to compare it to our current snapshot and make sure nothing unexpected has changed from before.

I have spent a lot of quality time with these tests, so if you want a second and third pair of eyes if the new prefix is creating issues, let me know!